ui/packages/shared/profile/src/ProfileMetricsGraph/index.tsx

```diff
@@ -13,22 +13,23 @@
 
 import {useState, useEffect} from 'react';
 import MetricsGraph from '../MetricsGraph';
-import {ProfileSelection, SingleProfileSelection} from '..';
-import {QueryServiceClient, QueryRangeResponse, Label, Timestamp} from '@parca/client';
+import {MergedProfileSelection, ProfileSelection} from '..';
+import {QueryServiceClient, QueryRangeResponse, Label, Timestamp, Duration} from '@parca/client';
 import {RpcError} from '@protobuf-ts/runtime-rpc';
 import {DateTimeRange, useGrpcMetadata, useParcaContext} from '@parca/components';
 import {Query} from '@parca/parser';
 import useDelayedLoader from '../useDelayedLoader';
+import {getStepDuration} from '@parca/functions';
 
 interface ProfileMetricsGraphProps {
   queryClient: QueryServiceClient;
   queryExpression: string;
   profile: ProfileSelection | null;
   from: number;
   to: number;
-  select: (source: ProfileSelection) => void;
   setTimeRange: (range: DateTimeRange) => void;
   addLabelMatcher: (key: string, value: string) => void;
+  onPointClick: (timestamp: any, labels: any, queryExpression: string) => void;
```

Comment:
If we can add the appropriate types to the props here, that'd be great.

Reply:
true, those anys were meant to be temporary! I'll change that